staging/src/k8s.io/component-helpers/README.md

```diff
@@ -0,0 +1,27 @@
+# component-helpers
+
+This repository provides helpers primarily for core components (core components as described in [Create a k8s.io/component-base repo](https://github.com/kubernetes/enhancements/blob/master/keps/sig-cluster-lifecycle/wgs/0032-create-a-k8s-io-component-repo.md#component-definition)) which are required by at least two separate binaries in kubernetes org.
+Yet, still with a high level of abstraction.
+
+The only allowed kubernetes dependencies are `k8s.io/apimachinery`, `k8s.io/api` and `k8s.io/client-go`.
```

Comment:
Could you please describe how this is different from component-base here?

Reply:
Done